src/Castle.Core.Tests/BugsReported/DynProxy230.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2004-2010 Castle Project - http://www.castleproject.org/
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.DynamicProxy.Tests.BugsReported
+{
+    using System;
+
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class DynProxy230 : BasePEVerifyTestCase
```

Comment:
Could you please move this test into `src/Castle.Core.Tests/Mixins` with a descriptive name, that `BugsReported` directory has tests named after old JIRA issues from way back.